app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+def copy_file(command: str) -> None:
+    try:
+        parts = command.split()
+
+        if (
+                len(parts) != 3
```

Comment:
I suggest to refactor this:
1. Try block should be covering only operations with file - OSError should be used in as an exception we are trying to catch. So no need to reraise OSError - you may catch it and print a message that error occured during file handling.
2. If statement should be rewritten backwards - so you don't have to create extra return instruction. So if all conditions are met - execute the rest of code, if not then script will exit 
3. Use unpacking to get `source_file` and `destination_file` - it would be more pythonic approach